lib/zendesk_apps_tools/command.rb

```diff
@@ -60,8 +64,18 @@ def new
 
       prompt_new_app_dir
 
-      directory_options = @iframe_location != 'assets/iframe.html' ? { exclude_pattern: /iframe.html/ } : {}
+      directory_options = {}
+      if options[:scaffold]
+        directory_options = { exclude_pattern: /^((?!manifest.json).)*$/ }
```

Comment:
If we're using the scaffold then we shouldn't use the template at all, no?

Reply:
Oh I see, this is meant to only grab the manifest.

Reply:
👍

Reply:
- disable the iframe prompt for --scaffold
- implement include_pattern

Reply:
`include_pattern` config is not implemented in [Thor::Actions#directory](https://github.com/erikhuda/thor/blob/master/lib/thor/actions/directory.rb).
Keep using `exclude_pattern` for now

Reply:
@yu-tian113 could you add a comment above this line?
# excludes everything but manifest.json

Reply:
👍 Done, thanks @danielbreves